Prevent nash-hotplug to keep running after boot using 100% of cpu

diff -rud mkinitrd-6.0.34-orig/nash/hotplug.c mkinitrd-6.0.34/nash/hotplug.c
--- mkinitrd-6.0.34-orig/nash/hotplug.c	2008-03-06 21:20:34.000000000 -0300
+++ mkinitrd-6.0.34/nash/hotplug.c	2008-03-12 18:48:46.000000000 -0300
@@ -547,7 +547,7 @@
                     nashLogger(nc, NASH_ERROR, "parent exited without telling us\n");
                     close(nc->hp_childfd);
                     nc->hp_childfd = -1;
-                    break;
+                    exit(1);
                 }
                 count += rc;
             }
